client/app/components/DirectoryContent.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import FileRow from './FileRow';
+import DirectoryRow from './DirectoryRow';
+
+const DirectoryContent = (props) => {
+  let files = [];
+  let directories = [];
```

Comment:
It's more of stylistic comment but would suggest doing `const files = props.directoryEntry.filter(...).sort()`